tests/test-cases/Group6-VIC-Machine/6-03-Delete.robot

```diff
@@ -95,3 +104,128 @@ Attach Disks and Delete VCH
+
+    # Verify VM exists
+    ${rc}  ${output}=  Run And Return Rc And Output  govc ls /ha-datacenter/vm/${dummyvm}
+    Log  ${output}
+    Should Contain  ${output}  ${dummyvm}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  govc version
+    Log To Console  ${output}
+
+    # Create temp RP
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.create "%{TEST_RESOURCE}/rp-${rand}"
+    Should Be Equal As Integers  ${rc}  0
+
+    # Move VCH to temp RP
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.migrate -pool "%{TEST_RESOURCE}/rp-${rand}" %{VCH-NAME}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Delete with force
+    ${moid}=  Get VM Moid  %{VCH-NAME}
+    ${ret}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux delete --target %{TEST_URL} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE} --id ${moid} --force
+    Log  ${output}
+    Should Contain  ${output}  Completed successfully
+
+    # Verify VM exists
+    ${rc}  ${output}=  Run And Return Rc And Output  govc ls /ha-datacenter/vm/${dummyvm}
+    Log  ${output}
+    Should Contain  ${output}  ${dummyvm}
+
+    # Delete VM and RP
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.destroy ${dummyvm}
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.destroy "%{TEST_RESOURCE}/%{VCH-NAME}"
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${output}=  Run And Return Rc And Output  govc pool.destroy "%{TEST_RESOURCE}/temp-%{VCH-NAME}"
+    Log  ${output}
+    Should Be Equal As Integers  ${rc}  0
+
+    Run Keyword And Ignore Error  Cleanup VCH Bridge Network  %{VCH-NAME}
+
+
+Delete VCH moved to root RP and original RP deleted
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Pass Execution  Test skipped on ESX due to unable to move into RP
+    ${rand}=  Generate Random String  15
+    ${dummyvm}=  Set Variable  anothervm-${rand}
+    Set Suite Variable  ${tempvm}  ${dummyvm}
+    Log To Console  Create VM ${dummyvm} in %{TEST_RESOURCE}/%{VCH-NAME} net %{PUBLIC_NETWORK}
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.create -pool="%{TEST_RESOURCE}" -net=%{PUBLIC_NETWORK} -on=false ${dummyvm}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Verify VM exists
+    ${rc}  ${output}=  Run And Return Rc And Output  govc ls /ha-datacenter/vm/${dummyvm}
+    Log  ${output}
+    Should Contain  ${output}  ${dummyvm}
+
+    # Move VCH to root RP
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.migrate -pool %{TEST_RESOURCE} %{VCH-NAME}
+    Should Be Equal As Integers  ${rc}  0
+
+    # Delete with force
+    ${moid}=  Get VM Moid  %{VCH-NAME}
+    ${ret}  ${output}=  Run And Return Rc And Output  bin/vic-machine-linux delete --target %{TEST_URL} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource=%{TEST_RESOURCE} --id ${moid} --force
```

Comment:
As a note, if you're specifying `--id` then I don't think you need `--compute-resource`. I frequently use `--id` to override which VCH I'm deleting when working with nimbus or vmc.